### PR TITLE
Answer:37 - Biglist

### PR DESCRIPTION
**Pull request by jdegand**
Not really sure how many items you wanted displayed but I took the hint of 20 from the readme.  

Used minBufferPx / maxBufferPx to improve performance even more.  
 
Not much info about using trackBy with cdk-virtual-scroll-viewport.  The docs mentioned you just do it like you normally would.

### CODE REVIEW

File: apps/performance/ngfor-biglist/src/app/person-list.component.ts [hunk 1]
```diff
@@ -27,4 +31,8 @@ import { Person } from './person.model';
 })
 export class PersonListComponent {
   @Input() persons: Person[] = [];
+
+  trackPerson(index: number, person: Person) {
+    return person ? person.email : undefined;
```
Comment (tomalaforge) [inline apps/performance/ngfor-biglist/src/app/person-list.component.ts:36]: question: why did you do this ternary statement? Can person be undefined?

File: apps/performance/ngfor-biglist/project.json [hunk 2]
```diff
@@ -19,8 +19,12 @@
...
-        "scripts": []
+        "styles": [
+          "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
+          "apps/performance/ngfor-biglist/src/styles.scss"
+        ],
+        "scripts": [],
+        "allowedCommonJsDependencies": ["seedrandom"]
       },
       "configurations": {
         "production": {
```
Comment (tomalaforge) [inline apps/performance/ngfor-biglist/project.json:30]: You can rebase to update this file 👍 Thanks for your commit

File: apps/performance/ngfor-biglist/src/app/person-list.component.ts (hunk 1 above)
Comment (jdegand) [reply apps/performance/ngfor-biglist/src/app/person-list.component.ts]: Copy & Pasted from another project.  I forgot to add the index parameter so I just grabbed a similar trackBy I had recently used.

Comment (tomalaforge) [reply apps/performance/ngfor-biglist/src/app/person-list.component.ts]: You can also use my trackBy directive that simplify the work. `libs/shared/directives/src/lib/track-by.directive.ts`

**Pull request closed without merging**